### ISSUE TITLE
Pixel C and RCM-mode

### ISSUE THREAD

**Issue by 24mu13** [fail0verflow/shofel2#27]
I got the exploit is working (by definition probably) on Pixel C too https://github.com/fail0verflow/shofel2/issues/22.

Could I ask how to put the Pixel C on RCM-equivalent mode? Is it simply the fastboot mode?
Excuse my stupid question, but I cannot find anything on [README](https://github.com/fail0verflow/shofel2/blob/master/README.md) or on the original article.

**Comment by Masamune3210** [comment]
Just because you got into rcm doesn't mean it's susceptible to the exploit

**Comment by Masamune3210** [comment]
There isn't much point on a phone anyway if it has a unlocked bootloader

**Comment by 24mu13** [comment]
> Just because you got into rcm doesn't mean it's susceptible to the exploit

Yes it is, as stated by the author here https://github.com/fail0verflow/shofel2/issues/22 and in the [original article](https://fail0verflow.com/blog/2018/shofel2/); makes actually a lot of sense also to me, since the chipset is the same of the Nintendo Switch, i.e. Tegra X1.

> There isn't much point on a phone anyway if it has a unlocked bootloader

Yes there is, because Pixel C is susceptible to a brick situation: even with unlocked bootloader you can't flash anything unless you set a specific NVM flag; this operation cannot be done without a working recovery (you need to run it using **adb** shell `fwtool vbnv write dev_boot_fastboot_full_cap 1`). In brief, with recovery partition erased and without exploit you can throw away your Pixel C.

**Comment by Masamune3210** [comment]
I stand corrected, I didn't realize there was a bricking issue with the C, I apologize

**Comment by shuffle2** [comment]
To enter RCM on pixel c i did the soft reset sequence (poking some registers to reboot into rcm), instead of using hardware button combo. Maybe there is a hardware button combo but I don't know it.

**Comment by 24mu13** [comment]
> To enter RCM on pixel c i did the soft reset sequence (poking some registers to reboot into rcm), instead of using hardware button combo. Maybe there is a hardware button combo but I don't know it.

Thank you very much for the answer. I am afraid that's the **recovery mode** (using Android terminology). I have a situation where I am unable to boot or flash any recovery: that's why I hoped to use your exploit from the **fastboot mode**.

**Comment by shuffle2** [comment]
You might have to open the device
